utils/devel/sync-fork.sh

```diff
@@ -42,14 +44,15 @@ if ! [[ $AvailableRemotes =~ $Upstream ]]; then
     git remote add upstream "$Upstream"
 fi
 
-git fetch upstream || Error "Fetching Upstream."
-
+CurrentBranch=`git rev-parse --abbrev-ref HEAD`
 
 git checkout master || Error "Checking out Master."
 
+git fetch upstream || Error "Fetching Upstream."
 
 git merge upstream/master || Error "Merging upstream/master with local /master"
```

Comment:
@sbts I think this merges into the active branch, not into the master branch, right?